.golangci.yaml

```diff
@@ -33,5 +33,14 @@ linters:
     # ./scripts/check_license.sh is run by ./scripts/presubmit.sh
 
 issues:
-    # Don't turn off any checks by default. We can do this explicitly if needed.
-    exclude-use-default: false
+  # Don't turn off any checks by default. We can do this explicitly if needed.
+  exclude-use-default: false
+  # List of regexps of issue texts to exclude.
+  #
+  # But independently of this option we use default exclude patterns,
+  # it can be disabled by `exclude-use-default: false`.
+  # To list all excluded by default patterns execute `golangci-lint run --help`
+  #
+  # Default: https://golangci-lint.run/usage/false-positives/#default-exclusions
+  exclude:
+    - "package-comments: should have a package comment"
```

Comment:
Should this change be in this pr (I think there's a separate PR with something similar)?

Reply:
That's right. I plan to rebase this PR when another PR is approved and merged, so we will have two commit history.